src/components/sections/compensation-calculator/types.ts

```diff
@@ -0,0 +1,9 @@
+import { SalariesByLocation } from "studio/lib/interfaces/compensations";
+
+export type CompensationData = {
+  slug: string;
+  salariesByLocation: SalariesByLocation[];
+};
+export type Degree = "bachelor" | "master";
```

Comment:
Do we need "other" as well?

Reply:
I know this is a discussion, but it makes it harder to do calculations. I think actually the proper way forward is to instead only communicate "years of relevant experience" and avoid communication on bachelor/masters, but still this is often a requirement from customers so there is a tradeoff here.

Reply:
True! I'm not really that keen on going into this discussion again, so I guess this is fine by me.